### PR TITLE
Lesson 3: Add support for hidden (black) squares.

### PR DESCRIPTION
**Pull request by gpoesia**
These are positions the game camera cannot see.

### CODE REVIEW

File: src/lesson/lesson03.js [hunk 1]
```diff
@@ -248,13 +248,15 @@ function Lesson03ExerciseStepPlayer(rows, cols, robot_position,
                                     robot_direction,
                                     components_positions,
                                     machines_positions,
-                                    goal) {
+                                    goal,
+                                    hidden_positions) {
```
Comment (nildo) [inline src/lesson/lesson03.js]: In the steps, all the hidden positions come before the goal, if you could change the order of these two parameters would be easier than change all the steps.

**Comment by gpoesia** [comment]
@nildo I've rebased, made this change and removed the hidden positions from comments. Could you please check?

**Comment by nildo** [comment]
Ok, now it looks perfect.

**Comment by gpoesia** [comment]
All right, thanks! A pull request that enables the black squares is already coming along.

**Pull request merged**